app/react/components/Svg.tsx

```diff
@@ -53,6 +53,7 @@ import upload from '@/assets/ico/upload.svg?c';
 import url from '@/assets/ico/url.svg?c';
 import usercircle from '@/assets/ico/user-circle.svg?c';
 import userlock from '@/assets/ico/user-lock.svg?c';
+import kubernetes2 from '@/assets/ico/kubernetes-2.svg?c';
```

Comment:
kubernetes2? do we have another kubernetes svg?

Reply:
I brought this over from EE, expecting it to be missing in CE. But on searching, I found we have kube.svg in CE, and kubernetes-2.svg in EE. 

I have updated CE and EE to only use kube.svg. Thanks for the spot :)